src/fault.h

```diff
@@ -61,6 +69,8 @@ class fault
         fault_id id_;
         translation name_;
         translation description_;
+        translation adds_fault_;
+        translation flags_;
```

Comment:
So you want those two lines deleted?

Reply:
It would still be compatible since this solution does not allow cleaning and lubricating at once. You'd still need the lubricate function in case people choose NOT to lubricate.

Reply:
I mean, I'm going to change the structure of `class fault` to support multiple mending actions for a single fault, which would cause merge conflict with your new changes, so I want you to temporarily hold off these changes until I finish my implementation, and then you can implement the full feature of your PR (cleaning with and without lubricating) using the new format I'm going to implement.

The comment about `translation` is another issue. `translation`, as the name suggests, translate a string into other languages, but since `adds_fault_` and `flags_` here are internal strings, which should be kept the same no matter what language the game is using. It has nothing todo with compatibility, I just wanted you to know that `translation` shouldn't be used like this.